main.go

```diff
@@ -187,10 +188,14 @@ func toolVersion(tool string) (string, error) {
 	return out, nil
 }
 
-func findByExt(root, ext string) ([]string, error) {
+func findBuildArtifact(rootDir, artifactExt string, buildStartTime time.Time) ([]string, error) {
```

Comment:
function name: `findBuildArtifact` --> `findArtifact`
parameter names:
- why `artifactExt` is better than `ext`?
- `buildStartTime` --> `buildStart` or `build`, no need for include the type in the var name.

Prefer commenting the function over long parameter and function names.